packages/openneuro-app/src/scripts/utils/global-polyfill.ts

```diff
@@ -1,16 +1,16 @@
-/* eslint-disable */
-// Workaround for incorrect global reference in draft.js via fbjs
-interface Object {
-  // eslint-disable-next-line @typescript-eslint/ban-types
-  global: object
-}
+// /* eslint-disable */
+// // Workaround for incorrect global reference in draft.js via fbjs
+// interface Object {
+//   // eslint-disable-next-line @typescript-eslint/ban-types
+//   global: object
+// }
 
-function polyfillGlobal(): void {
-  if (typeof global === "object") return
-  Object.defineProperty(Object.prototype, "global", {
-    get: () => globalThis,
-    configurable: true,
-  })
-}
+// function polyfillGlobal(): void {
+//   if (typeof global === "object") return
+//   Object.defineProperty(Object.prototype, "global", {
+//     get: () => globalThis,
+//     configurable: true,
+//   })
+// }
 
-polyfillGlobal()
+// polyfillGlobal()
```

Comment:
Glad we found this. I'd just remove this file from the source if this PR is cleaning up all the draft.js components anyways.